algorithms/number_palindrome.py

```diff
@@ -0,0 +1,18 @@
+def is_num_palindrome(num: int):
+    binary = []
+    is_converted = False
+
+    while not is_converted:
+        num, rem = divmod(num, 2)
+        binary.append(rem)
+        if num == 0:
```

Comment:
it's for `num = 0` corner-case?)

Reply:
it means we're done according to the steps i found on the internet (`Repeat the steps until the quotient is equal to 0.`)